strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-yum-layout-provider/pom.xml

```
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>strongbox-storage-layout-providers</artifactId>
+        <groupId>org.carlspring.strongbox</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>strongbox-storage-yum-layout-provider</artifactId>
```

Cosmetic: I know this is typically not required by Maven, but due to the release plugin that we're using, could you please explicitly specify the `groupId` and `version` of the project here:
```suggestion

            <groupId>org.carlspring.strongbox</groupId>
            <artifactId>strongbox-storage-yum-layout-provider</artifactId>
            <version>1.0-SNAPSHOT</version>

```

Thanks! :)